### PR TITLE
[chore] Add script to deploy CapabilityRegistry with hardcoded config

### PR DESCRIPTION
**Pull request by cedric-cordenier**

### CODE REVIEW

**Review by ettec — DISMISSED** [review]

File: core/scripts/keystone/05_deploy_and_initialize_capability_registry-sample.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (DeividasK) [inline core/scripts/keystone/05_deploy_and_initialize_capability_registry-sample.sh]: Nit: file name should be `capabilities_registry`

File: core/scripts/keystone/src/05_deploy_initialize_capability_registry.go [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+	if err != nil {
+		return kcr.CapabilitiesRegistryNodeParams{}, fmt.Errorf("failed to convert signer: %w", err)
+	}
+
+	var sigb [32]byte
+	copy(sigb[:], signerB)
+
+	return kcr.CapabilitiesRegistryNodeParams{
+		NodeOperatorId: nopID,
+		P2pId:          [32]byte(peerIDB),
```
Comment (DeividasK) [inline core/scripts/keystone/src/05_deploy_initialize_capability_registry.go]: Lint error

File: core/scripts/keystone/src/05_deploy_initialize_capability_registry.go [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+		Signer:         sigb,
+	}, nil
+}
+
+// Run expects the follow environment variables to be set:
```
Comment (DeividasK) [inline core/scripts/keystone/src/05_deploy_initialize_capability_registry.go]:
```suggestion
// Run expects the following environment variables to be set:
```

Comment (DeividasK) [reply core/scripts/keystone/src/05_deploy_initialize_capability_registry.go]: Missing vars?

**Review by ettec — APPROVED** [review]

**Review by DeividasK — APPROVED** [review]

**Pull request merged**